appsso-starter-java/config/workload.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     - name: appsso-starter-java
       ref:
         apiVersion: services.apps.tanzu.vmware.com/v1alpha1
-        kind: ResourceClaim
+        kind: ClassClaim
```

Comment:
Are there any documentation updates that need to go with this change from creating a ResourceClaim to a ClassClaim?

Reply:
The docs link on the README.md is still pointing to the 1.5 docs, should update that to the corresponding 1.6 doc along with this change

Reply:
Should we instead restore the readme instead of dealing with never-ending docs redirects?

Reply:
docs are in progress for the whole classclaim/resourceclaim difference for 1.6, we're trying to make sure we stay ahead of feature freeze here.

Reply:
@mamachanko and I updated the README.md here. Let me know if this addresses the concern.